vidpub/generate_playlist.py

```diff
@@ -0,0 +1,138 @@
+import datetime
+
+import json
+import os
+
+import requests
+
+
+from apiclient.discovery import build
+
+from slugify import slugify
+
+# Video data generator variables
+FIRST_DATE = datetime.date(
+    int(os.environ["YEAR"]), int(os.environ["MONTH"]), int(os.environ["DAY"])
+)
+
+YOUTUBE_API_KEY = os.environ.get("YOUTUBE_API_KEY", "")
+CHANNEL_ID = os.environ.get("CHANNEL_ID", "")
+PLAYLIST_TITLE = os.environ.get("PLAYLIST_TITLE", "")
+PLAYLIST_ID = os.environ.get("PLAYLIST_ID", "")
+CONFERENCE_DAY1 = os.environ.get("CONFERENCE_DAY1", "")
```

Comment:
I don’t think it’s permitted for any of these to be empty, so they should use `os.environ[key]` instead of  `get`.

Reply:
Yeah, most of the variables should use `os.environ[key]`. But one of `PLAYLIST_TITLE` and `PLAYLIST_ID` variable could be empty. I've updated the variables which can not be empty.